ee/server/service/software.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"context"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func (svc Service) ListSoftware(ctx context.Context, opts fleet.SoftwareListOptions) ([]fleet.Software, error) {
+	opts.IncludeCVEScores = true
+	return svc.Service.ListSoftware(ctx, opts)
+}
+
+func (svc *Service) SoftwareByID(ctx context.Context, id uint, includeCVEScores bool) (*fleet.Software, error) {
+	return svc.Service.SoftwareByID(ctx, id, true)
```

Comment:
Same as above, either `includeCVEScores` is not required, or is missing from the method call